owncloudApp/src/main/java/com/owncloud/android/presentation/sharing/shares/PublicShareDialogFragment.kt

```diff
@@ -321,6 +343,111 @@ class PublicShareDialogFragment : DialogFragment() {
         })
     }
 
+    private fun requirementsPasswordPolicy(password: String, passwordPolicy: OCCapability.PasswordPolicy) {
```

Comment:
```suggestion
    private fun updateRequirementsPasswordPolicy(password: String, passwordPolicy: OCCapability.PasswordPolicy) {
```
`requirementsPasswordPolicy` doesn't give a clue on what the method is used for, and the name should be self-documenting

Reply:
Changed